File: C#/Java/Pattern.java
import java.util.Scanner;
import java.io.*;
public class Pattern{
    public static void main(String[] args){
        Scanner scr = new Scanner(System.in);
        int x = scr.nextInt();
        for(int i=1;i<=x; i++){
            for(int j=1; j<=i; j++)
            System.out.print(" * ");
            System.out.println(" ");
        }     
        System.out.println("Another type of pattern:");
        for(int i = 1; i<=x; i++){
            for(int j = x; j>=i; j--)
            System.out.print(" * ");
            System.out.println(" ");
        } 
    }
}

File: Kotlin/pattern.kt
fun main(){
    for(i in 1..6){
        for(j in 1..i){
            print(" * ")
        }
        println(" ")
    }
    println("Another type of Pattern")
    for(i in 1..6){
        for(j in i..6){
            print(" * ")
        }
        println(" ")
    }
}